libraries/matrix/test/src/main/kotlin/io/element/android/libraries/matrix/test/room/FakeMatrixRoom.kt

```diff
@@ -91,6 +90,8 @@ class FakeMatrixRoom(
     canRedactOwn: Boolean = false,
     canRedactOther: Boolean = false,
 ) : MatrixRoom {
+    override val isOneToOne: Boolean = activeMemberCount <= 2L
```

Comment:
Do we really need to override the default value (which is `activeMemberCount == 2L`) here? It looks like this is against what the spec says.